pyproject.toml

```diff
@@ -102,6 +102,7 @@ azure-cognitiveservices-speech = {version = "^1.28.0", optional = true}
 py-trello = {version = "^0.19.0", optional = true}
 momento = {version = "^1.5.0", optional = true}
 bibtexparser = {version = "^1.4.0", optional = true}
+wasm-exec = "^0.1.7"
```

Comment:
this should not be required dependeyncy, lets make optional

Reply:
This seems dependent on the conversation below. Given that this is a big reason for the CVE that are flagging Langchain in different security scans, I'd advocate (for now) to make it a required dependency.